gnomad_qc/v5/sample_qc/hard_filters.py

```diff
@@ -88,7 +88,7 @@ def compute_aou_sample_qc(
 
 def main(args):
     """Determine samples that fail hard filtering thresholds."""
-    hl.init(tmp_dir=f"{WORKSPACE_BUCKET}/tmp/4_day")
+    hl.init(tmp_dir=f"gs://{WORKSPACE_BUCKET}/tmp/4_day")
```

Comment:
sorry I didn't notice this! quick follow-up, question, why does `WORKSPACE_BUCKET` need a `gs://` prefix when `AOU_BUCKET` does not? https://github.com/broadinstitute/gnomad_qc/blob/39e7c3c7e1767c25e27184c9a43e15c4780dff59/gnomad_qc/v5/resources/constants.py#L16

Reply:
maybe we should change `AOU_BUCKET` instead (remove the `gs://` to make it consistent with `WORKSPACE_BUCKET`, `GNOMAD_BUCKET` and `GNOMAD_TMP_BUCKET`)?

Reply:
I like dropping gs:// from AOU_BUCKET